docs/user/openstack/deploy_baremetal_workers.md

```diff
@@ -0,0 +1,188 @@
+                 
+                 ... other settings
+                 
+                 compute:
+                 - architecture: amd64
+                   hyperthreading: Enabled
+                   name: worker
+                   platform:
+                     openstack:
+                       type: <baremetalComputeFlavor>
+                   replicas: 3
+         
+                 ... other settings
+                 
+                 platform:
+                   openstack:
+                     computeFlavor: <vmComputeFlavorForMasters>
+                     machinesSubnet: <uuidOfPreprovisionedSubnet>
+
+- Run the openshift installer:
+
+        ./openshift-install create cluster --log-level debug
+        
+- Wait for installer to complete. 
+
+    If the installer times out waiting for the baremetal workers to complete booting,
+    restart the installation usin the appropriate *wait-for* command:
+    
+        ./openshit-install watier-for bootstrap-complete --log-level debug
+
+## Deploy cluster with vm workers only and add BM workers after 
+Initial cluster deployment is done with only 
+vm workers attached to the installer-provisioned network. The bare-metal
+workers are added after initial installation. bare-metal workers are
+attached to a preexisting network.
+
+- Requirements: 
+    - Cloud provideer is configure to allow creation of  OpenStack subnets that supports attaching vms and baremental servers to it. 
+    
+- Create install-config.yaml:
+    - Set compute.[worker].platform.openstack.type to the vm flavor which will be used by worker nodes.
+    - Set control-plane.platform.openstack.type to the vm flavor which will be used by the control-plane masters.
+    - Set platform.openstack.computeFlavor to the vm flavor which will be used by the control-plane masters.
+    
+        For example:
+ 
+                controlPlane:
+                   platform:
+                     openstack:
+                       type: <vmComputeFlavorForMasters>
+                 
+                 ... other settings
+                 
+                 compute:
+                 - architecture: amd64
+                   hyperthreading: Enabled
+                   name: worker
+                   platform:
+                     openstack:
+                       type: <vmComputeFlavorForWorkers>
+                   replicas: 3
+         
+                 ... other settings
+                 
+                 platform:
+                   openstack:
+                     computeFlavor: <vmComputeFlavorForMasters>
+
+- Run the openshift installer:
+
+        ./openshift-install create cluster --log-level debug
+        
+- Wait for installer to complete. 
+
+- Once the installation is complete deploy the baremetal workers using machinesets and the machine-api-operator
+    - Create baremetal worker machineset yaml file: *baremetalMachineSet.yaml*  
+        
+          <need to provide examle here> 
```

Comment:
You might as well choose not to describe how to add a machineset, as this is already [covered](https://github.com/openshift/openshift-docs/pull/21896/files) by the downstream docs